.gitignore

```diff
@@ -134,3 +134,5 @@ report.xml
 
 .pixi
 .imod_collector
+.teamcity
+LHM2016_v01vrz
```

Comment:
Are these needed?

Reply:
wel als je de unsa database in je git working copy directory heb staan, maar het is de vraag of dat handig is. Ik heb m daarbuiten en ook de imod_collector. maar waarvoor heb je .teamcity nodig eigenlijk?

Reply:
Removed both